src/cast-player.js

```diff
@@ -877,17 +877,25 @@ class CastPlayer extends BaseRemotePlayer {
         loadOptions.media.breaks = breaks;
       }
     }
-    if (this._playerConfig.sources.captions && this._playerConfig.sources.captions.length)
-      loadOptions.media.tracks = this._playerConfig.sources.captions.map((caption, index) => {
-        let newTrack;
-        newTrack = new chrome.cast.media.Track(index + 1, chrome.cast.media.TrackType.TEXT);
-        Utils.Object.mergeDeep(newTrack, {
-          trackContentId: caption.url,
-          name: caption.label,
-          language: caption.language
-        });
-        return newTrack;
+    if (this._playerConfig.sources.captions && this._playerConfig.sources.captions.length) {
```

Comment:
LGTM.
But I would export all the external captions handling to a separate method since `_getLoadOptions` start to  become unreadable.

Reply:
done